app/src/main/java/com/uc/mymovies/MainActivity.kt

```diff
@@ -5,25 +5,27 @@ import android.os.Bundle
 import android.util.Log
 import android.widget.ListAdapter
 import android.widget.ListView
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProviders
 import com.google.firebase.firestore.FirebaseFirestore
 import com.google.firebase.firestore.FirebaseFirestoreSettings
+import com.uc.mymovies.main.MainFragment
+import com.uc.mymovies.main.MainViewModel
 import com.uc.mymovies.service.MovieService
 
 class MainActivity : AppCompatActivity() {
-
-    var movieService: MovieService = MovieService()
-
-    init {
-        fetchMovies()
-    }
-
-    private fun fetchMovies() {
-       var result = movieService.fetchUserMovies("connorkeith")
-        Log.d("TEST-LOG: ", result.toString())
-    }
+    // create fragment for each layout in a new package
+     private lateinit var mainFragment: MainFragment
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+       // mainFragment = MainFragment.newInstance()
```

Comment:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.